src/logutil/logger.cc

```diff
@@ -42,15 +42,19 @@ void Logger::log(const LogLevel &level, const string &message) {
 }
 
 void Logger::LogDebug(const string &message) {
+  s=__PRETTY_FUNCTION__;
   Logger::log(LogLevel::DEBUG, message);
 }
 void Logger::LogInfo(const string &message) {
+  s=__PRETTY_FUNCTION__;
   Logger::log(LogLevel::INFO, message);
 }
 void Logger::LogWarn(const string &message) {
+  s=__PRETTY_FUNCTION__;
```

Comment:
If you run this here, the _PRETTY_FUNCTION__ will tell you that this is Logger::LogWarn. This is not what we want right? We want to know where I used Logger::LogWarn.